### ISSUE TITLE
Preprocessing and performance on ModelNet40

### ISSUE THREAD

**Issue by yueqiw** [fxia22/pointnet.pytorch#41]
Hi,

We run the classification model on ModelNet40 dataset (w/o feature transform), but couldn't get > 0.8 accuracy even after 200 epochs (we could reproduce the shapenet result, though). So I'm asking for some suggestions. 

(1) Did you train ModelNet40 with the same learning rate schedule as in train_classification.py? 

(2) We prepared the data loader by randomly subsetting ModelNet40 vertices. As mentioned in https://github.com/fxia22/pointnet.pytorch/issues/35#issuecomment-471158774, you used a data processing pipeline to sample points from the mesh. Could you provide a high-level overview of how it's done or point to a reference that describes ModelNet40 data processing? 

Thanks you very much!

**Comment by yueqiw** [comment]
We solved the problem by using this link for ModelNet40 dataset: https://shapenet.cs.stanford.edu/media/modelnet40_ply_hdf5_2048.zip

**Comment by notabigfish** [comment]
@yueqiw
hi, may I ask how did you convert the .h5 file to .ply file? Thanks!
